cmd/assets.go

```diff
@@ -13,17 +13,17 @@ import (
 
 func assetsCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "asset",
+		Use:     "assets",
 		Aliases: []string{"assets"},
 		Short:   "Manage assets",
 		Annotations: map[string]string{
 			"group:core": "true",
 		},
 		Example: heredoc.Doc(`
-			$ compass asset list
-			$ compass asset get
```

Comment:
We can use `view` as a command name here. and edit for post.

Reply:
`view` seems better than get. But should we use something like `upsert` or `add` instead for post, edit seems like making changes to something existing. wdyt @ravisuhag ??

Reply:
Sorry, just noticed that `upsert` has been deprecated, edit make sense